numpy/core/src/common/npy_ctypes.h

```diff
@@ -0,0 +1,46 @@
+#ifndef NPY_CTYPES_H
+#define NPY_CTYPES_H
+
+#include <Python.h>
+
+#include "npy_import.h"
+
+/*!
```

Comment:
Is the '!' markup?

Reply:
Yeah, I think it's doxygen. Copied from `npy_import`, but it seems we don't use it elsewhere - removing